saraho1123/currency-exchange
# Feature/form

## Is this a fix or a feature?
feature

## What is the change?
+ sets up functionality for a form.
+ input for user's current currency, amount they want to exchange, and the currency they want to change it into
+ on button click, state is updated in App and passed to ExchangeContainer. 
+ the user is routed to ExchangeContainer
+ ExchangeContainer will make the api call and manipulate the data from the call (not completed yet)
+ CurrencyCards has not been completed yet.

## Where should the reviewer start?
+ line 10 of App.js
+ line 2 of Form.js
+ line 1 of apiCalls.js

## How should this be tested?
+ clone down
+ cd into project
+ run `npm install` in terminal
+ run `npm start` in terminal
Form should appear. It can be filled out and button clicked, and user taken to `/currency-cards`, but nothing useful will yet be displayed.

## Relevant Tickets
+ #4 
+ #3
